test/Entropy.FunctionalTests/E2ETestBase.cs

```diff
@@ -24,57 +24,75 @@ protected E2ETestBase(ITestOutputHelper output, string siteName) : base(output)
+        [InlineData(ServerType.WebListener, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Standalone)]
+        [InlineData(ServerType.WebListener, RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, ApplicationType.Standalone,
+            Skip = "https://github.com/aspnet/Hosting/issues/949")]
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.Clr, RuntimeArchitecture.x64, ApplicationType.Standalone,
+            Skip = "Temporarily disable MVC functional tests on net461")]
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.Clr, RuntimeArchitecture.x86, ApplicationType.Standalone,
+            Skip = "https://github.com/aspnet/Hosting/issues/949")]
+        [InlineData(ServerType.WebListener, RuntimeFlavor.Clr, RuntimeArchitecture.x64, ApplicationType.Standalone,
+            Skip = "Temporarily disable MVC functional tests on net461")]
+        [InlineData(ServerType.WebListener, RuntimeFlavor.Clr, RuntimeArchitecture.x86, ApplicationType.Standalone,
             Skip = "https://github.com/aspnet/Hosting/issues/949")]
         public Task WindowsOS(
             ServerType serverType,
+            RuntimeFlavor runtimeFlavor,
             RuntimeArchitecture architecture,
             ApplicationType applicationType)
         {
-            return RunTestAsync(serverType, architecture, applicationType);
+            return RunTestAsync(serverType, runtimeFlavor, architecture, applicationType);
         }
 
         [ConditionalTheory]
         [OSSkipCondition(OperatingSystems.Windows)]
-        [InlineData(ServerType.Kestrel, RuntimeArchitecture.x64, ApplicationType.Portable)]
-        [InlineData(ServerType.Kestrel, RuntimeArchitecture.x64, ApplicationType.Standalone,
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Portable)]
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Standalone,
             Skip = "https://github.com/dotnet/cli/issues/6397")]
-        [InlineData(ServerType.Nginx, RuntimeArchitecture.x64, ApplicationType.Portable)]
-        [InlineData(ServerType.Nginx, RuntimeArchitecture.x64, ApplicationType.Standalone,
+        [InlineData(ServerType.Nginx, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Portable)]
+        [InlineData(ServerType.Nginx, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Standalone,
             Skip = "https://github.com/dotnet/cli/issues/6397")]
         public Task NonWindowsOS(
             ServerType serverType,
+            RuntimeFlavor runtimeFlavor,
             RuntimeArchitecture architecture,
             ApplicationType applicationType)
         {
-            return RunTestAsync(serverType, architecture, applicationType);
+            return RunTestAsync(serverType, runtimeFlavor, architecture, applicationType);
         }
 
+#elif NET461
+#else
+#error Target framework needs to be updated
```

Comment:
The test was like this before the net46 target was removed. This change is just restoring the removed code, and changing the condition from NET46 to NET461